### ISSUE TITLE
Does lightkeeper support scheduling tasks?

### ISSUE THREAD

**Issue by lukedoomer** [kalaksi/lightkeeper#16]
I would like to try lightkeeper if lightkeeper is able to run task periodically with cron format?

**Comment by kalaksi** [comment]
Not really, but I'd like to know what you're trying to achieve? Do you want to refresh the hosts periodically? Because that's actually a planned feature.

The area you highlighted is a status summary for monitoring so you can see errors and warnings quickly.
